homeassistant/components/tradfri/sensor.py

```diff
@@ -23,10 +24,36 @@
 
 
 @dataclass
-class TradfriSensorEntityDescription(SensorEntityDescription):
+class TradfriSensorEntityDescriptionMixin:
+    """Mixin for required keys."""
+
+    value: Callable[[Device], Any | None] = None  # type: ignore
+
+
+@dataclass
+class TradfriSensorEntityDescription(
+    TradfriSensorEntityDescriptionMixin,
+    SensorEntityDescription,
+):
     """Class describing Tradfri sensor entities."""
 
-    value: Callable[[TradfriSensor], Any | None] = None  # type: ignore
+
+SENSOR_DESCRIPTION_API = TradfriSensorEntityDescription(
+    device_class=SensorDeviceClass.AQI,
+    native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    key=SensorDeviceClass.AQI,
+    # The sensor returns 65535 if the fan is turned off
+    value=lambda device: None
```

Comment:
If the lambda spans more than one line we prefer to define a regular function at the module level and set the function as an attribute here in the description.